btcec/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/btcsuite/btcd/btcec/v2
+
+go 1.17
+
+require (
+	github.com/btcsuite/btcd v0.22.0-beta
```

Comment:
Seems odd to have this dependency here, it's only current use is to access `chainhash.DoubleHashB` in `btcec/example_test.go`.

Reply:
/shrug 

Can ultimately be removed when we make `chaincfg` its own package.